Arduino/slave_node_example/slave_node_example.ino

```diff
@@ -0,0 +1,74 @@
+/*
+Example of a slave node using the CanOpen library based on Canfestivino
+Attach a potentiometer to A0
+Attach a Servo to pin 9
+Attach a MCP-2515 board using pin 10 as CS (you can change this in CO_can_Arduino.cpp if you need to)
+Upload sketch and open Serial Monitor (made sure baud is set to 115200
+Run canopen_master_example.py on another system attached to canbus network
+ */
+
+
+
+#include "canfestival.h" //#TODO move canfesival/canfestivino to a subfolder or something
+#include "slave_node_example.h" //generated using objdictedit.py
+// These all need to be included because canfestival needs them <= according to original canfestivino
+// commenting out does not result in compile errors, but may cause runtime errors
+//#include <avr/io.h>
```

Comment:
Als we deze packages niet gebruiken dan kan je ze beter niet tonen

Reply:
Ik wilde ze er in laten omdat ik niet 100% zeker weet dat ze niet nodig zijn

Reply:
Ok misschien moeten we hiervoor een issue aanmaken dan?